pages/index.tsx

```diff
@@ -11,10 +11,12 @@ import Intro from '@/components/organisms/Intro';
 import StarsGrid from '@/components/molecules/StarsGrid';
 import SectionHeader from '@/components/atoms/SectionHeader';
 import Experience from '@/components/organisms/Experience';
-import { LucideGraduationCap } from 'lucide-react';
+import Education from '@/components/organisms/Education';
+import {Briefcase, LucideGraduationCap} from 'lucide-react';
```

Comment:
```suggestion
import {Briefcase, LucideGraduationCap} from 'lucide-react';
```

Briefcase возьми из heroicons